make/validate.mk

```diff
@@ -8,4 +8,4 @@ $(DKP_BLOODHOUND_BIN):
 
 .PHONY: validate-manifests
 validate-manifests: $(DKP_BLOODHOUND_BIN)
-	$(DKP_BLOODHOUND_BIN) . --helm-repo-path ./helm-repositories
```

Comment:
Does this mean `HelmRepositories` are not validated (skipped) ?

Reply:
HelmRepositories are loaded by bloodhound to be able to render the Helm releases. That would fail if there are any errors.

The equivalent to this parameter in the .bloodhound.yml file is here:
https://github.com/mesosphere/dkp-catalog-applications/pull/69/files#diff-8b381ad8641ba0fa1aa3852ab82b3b53332148e9b7e7f3bc295b12ceb884640aR10

Reply:
In addition I now added the Flux Source CRDs, so the HelmRepositories can be validated against their Schema and no longer need to be skipped.